slf4j-bridge/src/main/scala/org/slf4j/impl/ZioLoggerFactory.scala

```diff
@@ -43,7 +43,13 @@ class ZioLoggerFactory extends ILoggerFactory {
   private[impl] def run(f: ZIO[Any, Nothing, Any]): Unit =
     if (runtime != null) {
       zio.Unsafe.unsafe { implicit u =>
-        runtime.unsafe.run(f)
+        runtime.unsafe.run {
+          val fiberRefs = Fiber.currentFiber().map(_.asInstanceOf[Fiber.Runtime[_, _]].unsafe.getFiberRefs())
+          fiberRefs match {
+            case Some(fiberRefs) => ZIO.setFiberRefs(fiberRefs) *> f
```

Comment:
This pattern of running a ZIO workflow just to log something is not very efficient. We can get the current loggers from the current `FiberRefs` and then we can just use those loggers to log directly without going through a ZIO workflow.

Reply:
yes, i think you are right, we can probably do it better, 
probably by using `Fiber.log`
but i propose to do it in next PR

Reply:
👍